### PR TITLE
Feature/database

### PR DESCRIPTION
**Pull request by ArtemisSoftware**

### CODE REVIEW

File: core/src/main/java/com/artemissoftware/core/data/database/entities/ReminderTimeEntity.kt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.artemissoftware.core.data.database.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+data class ReminderTimeEntity(
```
Comment (MKiperszmid) [inline core/src/main/java/com/artemissoftware/core/data/database/entities/ReminderTimeEntity.kt:7]: What will be the use of this entity?

**Review by MKiperszmid — COMMENTED** [review]
Just one quetion

File: core/src/main/java/com/artemissoftware/core/data/database/entities/ReminderTimeEntity.kt (hunk 1 above)
Comment (ArtemisSoftware) [reply core/src/main/java/com/artemissoftware/core/data/database/entities/ReminderTimeEntity.kt:7]: I want to have the time for the notifications on a table
I will change name of the table to NotificationWarningEntity so its clearer.

Comment (ArtemisSoftware) [reply core/src/main/java/com/artemissoftware/core/data/database/entities/ReminderTimeEntity.kt:7]: I will also changed the name of one field in order to make clearer

**Pull request merged**